app/src/main/java/mozilla/lockbox/view/ItemDetailFragment.kt

```diff
@@ -32,9 +37,18 @@ class ItemDetailFragment : BackableFragment(), ItemDetailView {
         val view = inflater.inflate(R.layout.fragment_item_detail, container, false)
         setupBackable(view)
 
+        addressLayout = view!!.inputHostname
         return view
     }
 
+    override val addressLayoutClicks: Observable<Unit>
+        get() = addressLayout!!.clicks()
+
+    override fun openWebsite(hostname: String) {
```

Comment:
Implementation note: I would rather have this work as a `RouteAction` that gets passed thru the `Dispatcher`, both for metric-gathering purposes and because we may have other views that need to open outside websites later.

Reply:
external apps opened as a route action now